manifests/examples/azure/jessie.yml

```diff
@@ -23,7 +23,11 @@ volume:
     root:
       filesystem: ext4
       size: 10GiB
-packages: {}
+packages:
+  components:
+  - main
+  - contrib
+  - non-free
```

Comment:
It's mostly a matter of whether you'd like the images to eventually be endorsed by the Debian team as official. Other than that there really isn't a problem with this.

By the way: I can see you're part of the windows-azure github team. Will you be using bootstrap-vz to create the offical azure debian images?